src/batchUploader.ts

```diff
@@ -173,9 +173,12 @@ export class BatchUploader {
 
         let customFlags: SDKEventCustomFlags = {...event.CustomFlags};
         let integrationAttributes: IntegrationAttributes = _Store.integrationAttributes;
+        const integrationSpecificIds = getFeatureFlag(Constants.FeatureFlags.CaptureIntegrationSpecificIds) as boolean;
+        const integrationSpecificIdsV2 = getFeatureFlag(Constants.FeatureFlags.CaptureIntegrationSpecificIdsV2) as string || '';
+        const isIntegrationCaptureEnabled = (integrationSpecificIdsV2 ? integrationSpecificIdsV2 !== Constants.CaptureIntegrationSpecificIdsV2Modes.None : false) || (integrationSpecificIds === true);
```

Comment:
this ternary operator is a bit confusing.  

This is a tiny bit shorter, but feels congnitively less complex to me.  

```suggestion
        const isIntegrationCaptureEnabled = (integrationSpecificIdsV2 && integrationSpecificIdsV2 !== Constants.CaptureIntegrationSpecificIdsV2Modes.None) || integrationSpecificIds === true;;
```

What do you think?

Reply:
I agree, makes it more readable. I have updated the same for all the occurrences of `isIntegrationCaptureEnabled`